common/app/components/Nav/links.json

```diff
@@ -40,7 +40,7 @@
     "isReact": true
   },
   {
-    "content": "Shop",
-    "link": "/shop"
+    "content": "Donate",
+    "link": "https://www.freecodecamp.com/donate"
   }
```

Comment:
`donate` takes you to `localhost:3000/en/donate`. Is there a way to remove the `en` when moving to a new page? If so, that might allow me to use `/donate`.